packages/shared/sierra-client/src/HttpSierraClient.ts

```diff
@@ -110,6 +111,47 @@ export default class HttpSierraClient implements SierraClient {
     });
   }
 
+  async getDeletedRecordNumbers({
+    start,
+    end,
+  }: {
+    start?: Date;
+    end?: Date;
+  } = {}): Promise<APIResponse<number[]>> {
+    try {
+      const instance = await this.getInstance();
+      const startDateString = start?.toISOString().slice(0, 10);
+      const endDateString = (end ?? new Date()).toISOString().slice(0, 10);
+
+      const deletedDate = startDateString
+        ? `[${startDateString}, ${endDateString}]`
+        : undefined;
```

Comment:
If an end date isn't specified by the user, do we need to supply one here, or could we just drop in an empty string?

e.g. `GET /patrons?deletedDate=[2001-01-01,]`

Reply:
See above and/or the tests - it defaults to the current date. I know the Sierra API is (sometimes) OK with half-bounded intervals but I'd prefer to be explicit.